google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/models/Mutation.java

```diff
@@ -87,6 +88,23 @@ public static Mutation fromProtoUnsafe(List<com.google.bigtable.v2.Mutation> pro
     return mutation;
   }
 
+  /**
+   * Constructs a row mutation from an existing protobuf object.
+   *
+   * <p>Callers must ensure that the protobuf argument is not using serverside timestamps. The
+   * client assumes that all mutations are idempotent and will retry in case of transient errors.
+   * This can lead to row duplication.
+   *
+   * <p>When applied, the resulting Mutation object will ignore the project id and instance id in
+   * the table_name and instead apply the configuration in the client
+   */
+  @VisibleForTesting
```

Comment:
Why is this visible for testing? I thought it was needed by RowMutation?

Reply:
This was my overlook, I have removed the unneeded annotation. PTAL